docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: '1'
+
+services:
+  crush-url-db:
+    image: postgres:14.7
+    container_name: crush-url-db
+    ports:
+      - '5434:5432'
```

```suggestion
      - '5432:5432'
```

As far as I know, the default port is 5432, is there any reason to set it to 5434?

---

I think we don't need it!

---

@minsoo-web I reflected it in [ed9e5b7](https://github.com/spark-kr/crush-url/pull/12/commits/ed9e5b7a01145b15c9eb0a2ae0410c70008bf38d)!